x-pack/plugins/security_solution/public/timelines/components/timeline/unified_components/styles.tsx

```diff
@@ -122,11 +124,17 @@ export const StyledTimelineUnifiedDataTable = styled.div.attrs(({ className = ''
       rgba(221, 10, 115, 0.05) 10px
     );
   }
-  .udtTimeline .euiDataGridRow:has(.nonRawEvent) .euiDataGridRowCell--firstColumn {
-    ${({ theme }) => `border-left: 4px solid ${theme.eui.euiColorWarning};`}
+  .udtTimeline .euiDataGridRow:has(.nonRawEvent) {
+    .euiDataGridRowCell--firstColumn,
+    .euiDataGridRowCell--lastColumn {
+      ${({ theme }) => `border-left: 4px solid ${theme.eui.euiColorWarning};`}
```

Comment:
I think because of this both `building block` and non-buildingblock` events are getting highlighted by a left yellow border. Is this expected?

![grafik](https://github.com/elastic/kibana/assets/7485038/0670d2c2-d0a8-4b35-916e-dfdc01e0e335)

Reply:
Nope, just alerts, not regular events. Regular events have the gray border (that's also how it currently is if you check on main)